### PR TITLE
Add operations

### PR DESCRIPTION
**Pull request by haesleinhuepf**
I'm just adding a note to this future PR: After merging this, we might need to update clij-opencl-kernels because there was a typo in a cl file. It took me some time to figure out [how to](https://github.com/tj/git-extras/pull/80) do this:
```
git pull --recurse-submodules
git submodule update --remote --recursive
```

### CODE REVIEW

File: src/cleSetNonzeroPixelsToPixelindexKernel.cpp [hunk 1]
```diff
@@ -23,6 +23,12 @@ void SetNonzeroPixelsToPixelindexKernel::SetOutput(Object& x)
     this->AddObject(&x, "dst");
 }
 
+void SetNonzeroPixelsToPixelindexKernel::SetOffset(float x)
```
Comment (StRigaud) [inline src/cleSetNonzeroPixelsToPixelindexKernel.cpp:26]: the offset in the kernels is expecting an int, not a float, this can explain the failing

Comment (haesleinhuepf) [reply src/cleSetNonzeroPixelsToPixelindexKernel.cpp]: YES! 🙌 Thanks for spotting this :-)

**Comment by haesleinhuepf** [comment]
Hey @StRigaud ,

I'm running out of time and won't manage Connected Components today. I also need to finish some slides ;-)

Would you mind testing this branch on your system, before you merge it? I'm happy to fix issues if there are some.

Btw. I updated to the clij website. There is a CLIc category now which fits to this branch here:
https://clij.github.io/clij2-docs/reference__CLIc

Thanks!

Cheers,
Robert

**Comment by StRigaud** [comment]
Hi @haesleinhuepf, No worries.
After a quick look, it seems that 2 tests fails: 
- equal_constant_test 
- not_equal_constant_test

I will try to see if I can fix that before merging.

**Comment by StRigaud** [comment]
first limitation of the DefineDimensionality() method to set a 2d or 3d kernel.
We are using the tag name of the source which can change, same for the dst. This can be a source of copy/paste error.
Need to find an alternative but not urgent.

**Comment by StRigaud** [comment]
I dont delete the branch if you wanna add some more

**Pull request merged**